public/app/core/components/AppChrome/TopBar/SingleTopBar.tsx

```diff
@@ -77,11 +81,23 @@ export const SingleTopBar = memo(function SingleTopBar({
               tooltip={t('navigation.megamenu.open', 'Open menu')}
             >
               <Stack gap={0} alignItems="center">
-                <Branding.MenuLogo className={styles.img} />
-                <Icon size="sm" name="angle-down" />
+                <Icon name="bars" size="xl" />
               </Stack>
             </ToolbarButton>
           )}
+          {!menuDockedAndOpen && (
+            <Tooltip placement="bottom" content={homeNav?.text || 'Home'}>
```

Comment:
This seems to be the same code as the HomeLink component, can we use that one here instead?